changelog/_unreleased/2025-01-29-mark-import-export-internal.md

```diff
@@ -0,0 +1,31 @@
+---
+title: mark Import/Export internal
+issue: NEXT-40446
+---
+# Core
+* Added `Shopware\Core\Content\ImportExport\Event\ImportExportAfterProcessFinishedEvent.php`
```

Comment:
please use the class reference and not the file name and add a small sentence about its purpose